kuma/static/styles/components/compat-tables/bc-table.scss

```diff
@@ -0,0 +1,263 @@
+/*
+Table layout
+- uses display: block for smaller screen sizes;
+- requires .bc-table-{$num} modifier to display columns correctly
+********************************************************************** */
+
+.text-content table.bc-table { /* specificity to over-ride .text-content styles */
```

Comment:
Nit: Use `//` style comments for consistency?

Reply:
Using /* CSS comments */